### PR TITLE
New Google Drive Images

### PR DESCRIPTION
**Pull request by josesanchez1815**
Included new images for the https://github.com/nih-cfde/organization/blob/master/CommunicationManagementHelp.md#sharing-files

### CODE REVIEW

**Comment by ACharbonneau** [comment]
Hey @josesanchez1815 

Does github show you the branch conflicts on this? I'm honestly not sure if you can see them without admin access.

**Comment by josesanchez1815** [comment]
@ACharbonneau  Yes it does. I missed it Sorry!

**Comment by ACharbonneau** [comment]
No worries :) Do you know how to do conflict fixes in github? If not I can show you

**Comment by josesanchez1815** [comment]
Oh I figured it out based on some tutorials.

File: CommunicationManagementHelp.md [hunk 1]
```diff
@@ -242,11 +242,14 @@ the recipient will need to log in using the email they specified in their [onboa
 To copy the sharable link:
 - Right-click on the desired file and select `Share` OR click on the desired file and click the share button near the top right of the screen
 - Click 'WHO HAS ACCESS' in the bottom left corner of the dialogue box that appears
+
+<img src="/images/who-has-access.PNG" alt="Link Sharing Off" width="300"/>
```
Comment (ACharbonneau) [inline CommunicationManagementHelp.md:246]: This one is the one I noticed was out of date :)

**Comment by ACharbonneau** [comment]
Awesome! So, just FYI, it probably didn't have the conflict when you did the PR, because I think what happened is that you submitted this PR, and then noticed that you needed to merge in the one from yesterday. But merging in yesterdays changes meant that now this PR had a different base. Looks like it was an easy fix though, and I'm delighted you figured it out :)

**Comment by josesanchez1815** [comment]
Wow you hit it on the nail! That is exactly what happened

File: CommunicationManagementHelp.md (hunk 1 above)
Comment (josesanchez1815) [reply CommunicationManagementHelp.md:246]: @ACharbonneau  Sorry what exactly is the change. I think the procedure you laid out is still relevant for copying links...

Comment (ACharbonneau) [reply CommunicationManagementHelp.md:246]: so, yesterday when I went to get a link to share, the screen had changed to this: 
image

did it not change for everyone?

Comment (josesanchez1815) [reply CommunicationManagementHelp.md:246]: That's strange. But yes I still see the same images as those that exist in the repo already.

Comment (ACharbonneau) [reply CommunicationManagementHelp.md:246]: huh. Ok. It must be some kind of phased rollout of new features thing? So then let's merge in these changes, and if/when the view changes for people who aren't me, we'll update then :)

**Pull request merged**